### ISSUE TITLE
OSD not loading when turbo-links is enabled

### ISSUE THREAD

**Issue by harukizaemon** [cbeer/openseadragon-rails#17]
I have turbo-links enabled in my app and OSD doesnt load when moving between pages.  However, if I force a full-page refresh, voila!

Any suggestions?

**Comment by cbeer** [comment]
Presumably you're using turbolinks 5? It looks like our javascript is only compatible with turbolinks 4 right now: https://github.com/IIIF/openseadragon-rails/blob/master/app/assets/javascripts/openseadragon/rails.js#L8

We'd need to add something like e.g. https://github.com/projectblacklight/blacklight/blob/master/app/assets/javascripts/blacklight/core.js#L15-L26 to add compatibility for both turbolinks 4 + 5.

**Comment by harukizaemon** [comment]
Ahh! yes, nice.  I can make that change manually for now and try it out.

**Comment by harukizaemon** [comment]
OK "that" worked.  As a work-around, I changed my `app/assets/javascripts/openseadragon.js` to look like:

```
//= require openseadragon/openseadragon
// require openseadragon/rails

//= require openseadragon/jquery

(function($) {
    function initOpenSeadragon() {
        $('picture[data-openseadragon]').openseadragon();
    }

    $(document).on('turbolinks:load', initOpenSeadragon);
    $(document).ready(initOpenSeadragon);
})(jQuery);
```

Not a generic solution of course but gets me up and running.  Thank you!

**Comment by cbeer** [comment]
Fixed in v0.3.1.
